lib/src/protocol_instance.rs

```diff
@@ -195,11 +195,15 @@ fn get_blob_proof_type(
     proof_type: VerifierType,
     blob_proof_type_hint: BlobProofType,
 ) -> BlobProofType {
-    match proof_type {
-        VerifierType::None => blob_proof_type_hint,
-        VerifierType::SGX => BlobProofType::ProofOfCommitment,
-        VerifierType::SP1 => BlobProofType::ProofOfEquivalence,
-        VerifierType::RISC0 => BlobProofType::ProofOfEquivalence,
+    if cfg!(feature = "proof_of_equivalence") {
```

Comment:
why do we need this flag `proof_of_equivalence` if you can change the match statement and select the a certain arm? you can do
```
VerifierType::RISC0 => BlobProofType::KzgVersionedHash
```

Reply:
I think POE contract verify is not fully ready, and it's a all-or-none config, so no need to separate risc0 & sp1.